### ISSUE TITLE
MockComponent does not work with ViewChild

### ISSUE THREAD

**Issue by rafehi** [ike18t/ng-mocks#17]
Very neat library that's helping us find errors that were being hidden by `NO_ERRORS_SCHEMA`.

However, I've found that we're unable to use `MockComponent` with `ViewChild`:

``` javascript
@Component({
  selector: 'child',
  template: '<span>this is a child</span>'
})
export class ChildComponent {
  public someMethod() {    
  }
}

@Component({
  selector: 'parent',
  template: '<child></child>'
})
export class ParentComponent implements AfterViewInit {
  @ViewChild(ChildComponent) childComponent;

  ngAfterViewInit() {
    console.log(this.childComponent);     // this.childComponent is undefined here
    this.childComponent.someMethod();
  }
}

describe('test', () => {
  let component: ParentComponent;
  let fixture: ComponentFixture<ParentComponent>;  
  beforeEach(async(() => {        
    TestBed.configureTestingModule({
      declarations: [
        ParentComponent,
        MockComponent(ChildComponent)
      ], 
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ParentComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('test', () => {    
  });
});
```

It would be great if we could:
1. Use `MockComponent` in conjunction with `ViewChild`
2. Spy on component functions  through `MockComponent`

**Comment by rafehi** [comment]
I've created a PR to implement the above:
#20

**Comment by ike18t** [comment]
Your PR was released in version 6.2.2
cea5055
